### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by mohammadian-amin** [eugenelet/Meta-rPPG#13]
Hi 

thanks for codes sharing !

I installed requirements and run python train.py 

but I have bellow error 

RuntimeError: result type Float can't be cast to the desired output type Byte

![image](https://user-images.githubusercontent.com/83285581/117957360-dadf3380-b2ce-11eb-91f0-0a977abeac19.png)

Please advise me!

**Comment by fchinasso** [comment]
Same problem here

**Comment by yuhongjiu8888** [comment]
Same problem here

**Comment by revacosmic1** [comment]
Same problem here

**Comment by skarlett992** [comment]
Same problem here

**Comment by HuangYuKaiLI1031** [comment]
> Hi
> 
> thanks for codes sharing !
> 
> I installed requirements and run python train.py
> 
> but I have bellow error
> 
> RuntimeError: result type Float can't be cast to the desired output type Byte
> 
> ![image](https://user-images.githubusercontent.com/83285581/117957360-dadf3380-b2ce-11eb-91f0-0a977abeac19.png)
> 
> Please advise me!

You should upgrade the pytorch

**Comment by Rachelwrx** [comment]
Same problem here. but I have upgrade my Pytorch version to 1.9.0

**Comment by SunHaixin0324** [comment]
![image](https://user-images.githubusercontent.com/66905581/230014569-3f26ec7e-b8b4-45ac-83bf-4a948d77965b.png)
I think we can reverse the original code

**Comment by bunnykek** [comment]
> ![image](https://user-images.githubusercontent.com/66905581/230014569-3f26ec7e-b8b4-45ac-83bf-4a948d77965b.png) I think we can reverse the original code

Yeah it works after it. But you also have to change new method to reshape.
